### ISSUE TITLE
Carpet Fixes Setting optimizedBiomeAccess is not a valid rule

### ISSUE THREAD

**Issue by clementinise** [fxmorin/carpet-fixes#49]
Pretty self explanatory, with latest Carpet Fixes (v1.7.5)

Idk if I should have more logs?

**Comment by FxMorin** [comment]
That is not helpful at all, not a valid what, rule? Did it auto-complete when doing /carpet-fixes optimized*?

**Comment by clementinise** [comment]
Sorry it literraly says `Carpet Fixes Setting optimizedBiomeAccess is not a valid` without anything else behind it x)

```
[31m[22:04:16] [Server thread/ERROR]: [CM]: Carpet Fixes Setting optimizedBiomeAccess is not a valid - ignoring...[m
[22:04:16] [Server thread/INFO]: [CM]: loaded setting optimizedHypot as true from carpet-fixes.conf[m
[22:04:16] [Server thread/INFO]: [CM]: loaded setting optimizedFurnaces as true from carpet-fixes.conf[m
[22:04:16] [Server thread/INFO]: [CM]: loaded setting optimizedRounding as true from carpet-fixes.conf[m
[22:04:16] [Server thread/INFO]: [CM]: loaded setting optimizedPoweredRails as true from carpet-fixes.conf[m
[22:04:16] [Server thread/INFO]: [CM]: loaded setting optimizedRandom as true from carpet-fixes.conf[m
[22:04:16] [Server thread/INFO]: [CM]: loaded setting optimizedTicketManager as true from carpet-fixes.conf[m
[22:04:16] [Server thread/INFO]: [CM]: loaded setting optimizedRecipeManager as true from carpet-fixes.conf[m
```
And no it didn't autocomplete
![image](https://user-images.githubusercontent.com/35346472/163709584-8986ca40-13ea-4dde-803c-1968dd8cd0e8.png)

![image](https://user-images.githubusercontent.com/35346472/163709548-af898e0c-4e0d-44ce-8bcc-49174fb15573.png)

**Comment by FxMorin** [comment]
Can you try this in any of the v1.8.x versions?

**Comment by FxMorin** [comment]
Seems rule `optimizedBiomeAccess` was disabled for v1.7.x if you are playing in a version lower than 22w05a (a snapshot after 1.18.1)
Due to conflicts. Although if you want to use it for 1.18.1 you can use [this verison](https://github.com/fxmorin/carpet-fixes/releases/tag/v1.6.0)
